src/convert/default.jl

```diff
@@ -43,9 +43,9 @@ function rcopytype(s::Ptr{IntSxp})
     elseif anyna(s)
         DataArray{Float64}
```

Comment:
opps, this should be `DataArray{Int}`. Could you also change this while you are working on it?